### DISCUSSION TITLE
Proxy Mapping creation use "JsonPartialMatcher" instead of "JsonMatcher"

### DISCUSSION THREAD

**Discussion by bvogtAZ** [wiremock/WireMock.Net#1171] [Q&A]
Is there anyway to make WireMock.Net create the proxy mapping using "JsonPartialMatcher", instead of the default "JsonMatcher"?
`  "Request": {
    "Body": {
      "Matcher": {
        "IgnoreCase": true,
        "Name": "JsonMatcher",
`
In addition, I want to omit specific request fields from the saved proxy mapping. Is there a way to define the ProxyAndRecordSettings so certain request fields are not saved? This way, a "partial match" is performed?

**Comment by StefH** [comment]
@bvogtAZ 
1. I think this only applies to the Body, right?  I guess I can add this to ProxyAndRecordSetting.

2. I guess this is also possible.


I'll keep you updated if I make a preview release for your requests.

**Reply by bvogtAZ** [reply]
Correct, it only applies to the Body. ProxyAndRecordSettings sounds perfectly logical!
Thank you!
